src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/KeyboardInputActionObserver.cs

```diff
@@ -79,7 +79,7 @@ public class KeyboardInputActionObserver : MonoBehaviour
             { "p", Key.P },
             { "leftBracket", Key.LeftBracket },
             { "rightBracket", Key.RightBracket },
-            { "backSlash", Key.Backslash },
+            { "backslash", Key.Backslash },
```

Comment:
This is a fix for an error when simulating backslash key events (it is actually named "backslash" in the control path, rather than "backSlash").